berkeley-mobile/Calendar/Campus/Cells/CampusEventCollectionView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CampusEventCollectionView.swift
+//  berkeley-mobile
+//
+//  Created by Shawn Huang on 10/3/20.
+//  Copyright © 2020 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+class CampusEventCollectionView: UICollectionView {
+    
+    static let kCellIdentifier = "campusEventCollection"
+    
+    init(frame: CGRect) {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        layout.minimumLineSpacing = 13
+        layout.itemSize = CampusEventCollectionViewCell.kCardSize
+        
+        super.init(frame: frame, collectionViewLayout: layout)
+        
+        backgroundColor = .clear
+        layer.masksToBounds = false
+        showsVerticalScrollIndicator = false
+        showsHorizontalScrollIndicator = false
+        register(CampusEventCollectionViewCell.self, forCellWithReuseIdentifier: CampusEventCollectionView.kCellIdentifier)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+}
```

Comment:
This is pretty similar to `CardCollectionView`, with a different registered cell class + identifier. Is there a need to redefine it instead of using a `CardCollectionView` and registering a `CampusEventCollectionViewCell`? Curious to hear what you think.

Reply:
yeah makes sense, not likely to change this card anytime soon